jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -1724,8 +1821,12 @@ def dot_product_attention(
       q_seqlen = jnp.zeros(0, dtype=query.dtype)
     if kv_seqlen is None:
       kv_seqlen = jnp.zeros(0, dtype=query.dtype)
+    if q_offsets is None:
+      q_offsets = jnp.zeros(0, dtype=query.dtype)
```

Comment:
Do we have to do this? Can we pass `None` instead?

Also OOC why `zeros` and not `jnp.full(-1, ...)`?

Reply:
I think None is not a valid jax type? So what I do is to create a dummy tensor as a WAR. Another thing is that I want to add this tensor in nondiff_argnums but I think jax expects array to not be not included but return None in bwd to achieve the same. Do we have to do this or is a better way to achieve the same behavior?